Dockerfile

```diff
@@ -13,6 +14,13 @@ COPY --from=composer:latest /usr/bin/composer /usr/local/bin/composer
 RUN composer install --working-dir=/app --no-dev --prefer-dist \
     && rm  /usr/local/bin/composer
 
-WORKDIR /app
+WORKDIR /app/server/certs
+RUN openssl genrsa -des3 -passout pass:x -out server.pass.key 2048 && \
+  openssl rsa -passin pass:x -in server.pass.key -out server.key && \
+  rm server.pass.key && \
+  openssl req -new -key server.key -out server.csr -subj "/C=UK/ST=Warwickshire/L=Leamington/O=PDSInterop/OU=PDSInterop/CN=solid.pdsinterop.org" && \
```

Comment:
Although leaving in the original C/ST/L is great for finding the original post of the solution (I guess https://stackoverflow.com/a/44047595/153049 ?) we might want to use NL/Overijssel/Enschede as C/ST/L?